### PR TITLE
Initial commit: Timestream Kafka Sink Connector

### PR DESCRIPTION
**Pull request by kayalvizhi-kandasamy**
*Issue #, if available:*

*Description of changes:*
Initial commit for the Timestream Kafka Sink connector

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: integrations/kafka_connector/resources/kafka-connector-timestream-1.0-SNAPSHOT-jar-with-dependencies.jar
Comment (sethusrinivasan) [inline integrations/kafka_connector/resources/kafka-connector-timestream-1.0-SNAPSHOT-jar-with-dependencies.jar:1]: Pls exclude this file. Let us not checked in compiled code in repo

File: integrations/kafka_connector/src/main/java/software/amazon/timestream/utility/TimestreamWriter.java [hunk 1]
```diff
@@ -0,0 +1,313 @@
...
+    private final SchemaDefinition schemaDefinition;
+
+    /**
+     *
+     * @param schemaDefinition table schema
+     * @param config connector config
+     */
+    public TimestreamWriter(final SchemaDefinition schemaDefinition, final TimestreamSinkConnectorConfig config) {
+
+        this.databaseName = config.getDatabaseName();
```
Comment (sethusrinivasan) [inline integrations/kafka_connector/src/main/java/software/amazon/timestream/utility/TimestreamWriter.java:69]: Can you also capture the partition key on Timestram table? https://docs.aws.amazon.com/timestream/latest/developerguide/customer-defined-partition-keys.html and when you create the table ( may be during initialization ), you  can create with that specific partition key

File: integrations/kafka_connector/resources/kafka-connector-timestream-1.0-SNAPSHOT-jar-with-dependencies.jar
Comment (kayalvizhi-kandasamy) [reply integrations/kafka_connector/resources/kafka-connector-timestream-1.0-SNAPSHOT-jar-with-dependencies.jar:1]: Sure, removed it from the repo.

File: integrations/kafka_connector/src/main/java/software/amazon/timestream/utility/TimestreamWriter.java (hunk 1 above)
Comment (kayalvizhi-kandasamy) [reply integrations/kafka_connector/src/main/java/software/amazon/timestream/utility/TimestreamWriter.java:69]: Sure, created the Timestream table with partition key (as part of the CFT) and added a statement here in the log that mentions the partition key

**Pull request merged**